database/migrations/2017_04_21_162001_create_vouchers_table.php

```diff
@@ -15,20 +15,16 @@ public function up()
     {
         Schema::create('vouchers', function (Blueprint $table) {
             $table->increments('id'); // a voucher *instance*.
-            $table->integer('assigner_id')->unsigned()->nullable(); // who was assigned this voucher.
-            $table->integer('redeemer_id')->unsigned()->nullable(); // who redeemed this with
+            $table->integer('assignee_id')->unsigned()->nullable(); // who was assigned this voucher.
             $table->integer('creditor_id')->unsigned()->nullable(); // who's owed money for this.
             $table->string('code', 32); // the actual voucher code.
             $table->string('currentstate', 24)->default('requested');
             $table->integer('sponsor_id')->unsigned();  // the organisation that sponsored this token (usually an LA).
             $table->timestamps();
+            $table->softDeletes();
         });
 
         Schema::table('vouchers', function (Blueprint $table) {
-            $table->foreign('redeemer_id')
```

Comment:
Ignore assignee. I'll pull it. It's supposed to be an ID for a family that the Children's Centres give vouchers to. I may even pull it if next weds determines that we cant store that data in here.